### ISSUE TITLE
Ugly error message from adapter-mgmt

### ISSUE THREAD

**Issue by EmmEff** [UnivaCorporation/tortuga#73]
When attempting to create a resource adapter configuration profile without any values, the following error message is displayed:

```shell
[root@HOST ~]# adapter-mgmt create -r aws -p blah
Traceback (most recent call last):
  File "/opt/tortuga/bin/adapter-mgmt", line 509, in <module>
    AdapterMgmtCLI().run()
  File "/opt/tortuga/bin/adapter-mgmt", line 174, in run
    handler(args)
  File "/opt/tortuga/bin/adapter-mgmt", line 251, in create_resource_adapter_config
    cfg = [dict(key=key, value=value) for key, value in args.setting]
TypeError: 'NoneType' object is not iterable
```

**Comment by dtougas** [comment]
What should happen in this case? Should it create a blank profile? Or should it error out telling the user that some settings are required?

**Comment by dtougas** [comment]
Fix in PR #82.
